src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -222,7 +222,6 @@ public class FlowStateAllowedEventService {
                 STANDARD_DIRECTION_ORDER_DJ,
                 CREATE_SDO,
                 NotSuitable_SDO,
-                TAKE_CASE_OFFLINE,
```

Comment:
I dont think we should do this - this will prevent the scheduler from triggering the automatic "Take Case Offline" Event when a claimant solicitor fails to notify claim details and the deadline is surpassed.

Reply:
@sankhajuria can you please clarify why this has been removed?